packages/connectivity/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: connectivity_tizen
+description: Flutter plugin for discovering the state of the network (WiFi) connectivity on Tizen.
+version: 1.0.0
+homepage: https://github.com/flutter-tizen/plugins
+
+flutter:
+  plugin:
+    platforms:
+      tizen:
+        pluginClass: ConnectivityTizenPlugin
+        fileName: connectivity_tizen_plugin.h
+dependencies:
+  connectivity: ^0.4.9
+  connectivity_platform_interface: ^1.0.3
```

Comment:
The latest connectivity plugin version is 2.0.2 and the platform interface version is 1.0.6. Would it be better to update the dependencies before connectivity_tizen is published on pub.dev for the first time?

Reply:
You are right. It was updated with latest version.